Source/SPNetworkPreferencePane.m

```diff
@@ -163,14 +174,95 @@ - (IBAction)pickSSHClient:(id)sender
 	SPClear(_currentAlert);
 }
 
+- (IBAction)pickSSHConfig:(id)sender
+{
+	_currentFilePanel = [NSOpenPanel openPanel];
```

Comment:
It would be useful to specify a default file location for the file picker to open to. See the connection controller. I would prefer ~/.ssh perhaps?

Reply:
I also saw this notice in the log? 
`WARNING: <NSOpenPanel: 0x1012b4b40> running implicitly; please run panels using NSSavePanel rather than NSApplication.`

Reply:
I changed the default starting directory now to the .ssh directory of the current user.

The warning occurred because I started the NSOpenPanel as sheet modal of a non-existing window. After I changed it now to `beginWithCompletionHandler`, the warning won't show up anymore. Is this way sufficient?

Reply:
That should work as far as I know! Honestly much of my background is swift on iOS so the test and if it works it ships mentality generally applies to my objective C on Mac approach 😆